fcrepo-search-impl/src/main/java/org/fcrepo/search/impl/DbSearchIndexImpl.java

```diff
@@ -188,21 +199,26 @@ public void addUpdateIndex(final ResourceHeaders resourceHeaders) {
             try {
 
                 final var params = new MapSqlParameterSource();
-                params.addValue( fedoraIdParam, fullId);
+                params.addValue(fedoraIdParam, fullId);
                 final var modifiedParam = "modified";
+                final var contentSizeParam = "content_size";
+                final var mimetypeParam = "mime_type";
                 params.addValue(modifiedParam, new Timestamp(resourceHeaders.getLastModifiedDate().toEpochMilli()));
+                params.addValue(mimetypeParam, resourceHeaders.getMimeType());
+                params.addValue(contentSizeParam, resourceHeaders.getContentSize());
                 final String sql;
                 if (result.size() > 0) {
                     //update
                     sql = "UPDATE " + SIMPLE_SEARCH_TABLE +
-                            " SET modified=:" + modifiedParam + " WHERE fedora_id = :" + fedoraIdParam + ";";
+                            " SET modified=:" + modifiedParam + ", content_size=:" + contentSizeParam + ", " +
```

Comment:
This query is completely static, isn't it? Could also be a constant without any string concatenation.

Reply:
yes. definitely.